src/types/account-id.ts

```diff
@@ -62,7 +63,9 @@ class AccountID extends Hash160 {
    * @returns the base58 string defined by this.bytes
    */
   toBase58(): string {
-    return encodeAccountID(this.bytes);
+    /* eslint-disable @typescript-eslint/no-explicit-any */
+    return encodeAccountID(this.bytes as any);
```

Comment:
seems like we could make an improvement to `ripple-address-codec`?

Reply:
Lets discuss this one in person.